### DISCUSSION TITLE
Is isotope plugin now the only way to get masonry grid?

### DISCUSSION THREAD

**Discussion by hoops78** [bootscore/bootscore#735] [Q&A] [answered]
I'm revisiting a site I did a few years ago and I used the plugin Bs Masonry. As this is now archived. Do we now use isotope instead (which has the masonry feature)? I don't need any of the isotope features, so it might be a bit of an overkill. Would I be better off using masonry.pkgd.min.js?

**Comment by crftwrk** [comment, marked as answer]
> As this is now archived

Masonry plugin is not archived, it's packed now inside the loop templates .zip file https://github.com/bootscore/bs-loop-templates/releases/download/v5.3.2/bs-loop-templates-UnzipFirst.zip.

You can use it as usual in combination with the masonry loop templates https://bootscore.me/archives/masonry/.

Solved?

**Reply by hoops78** [reply]
Ah, I see, thank you.
